src/node_file.cc

```diff
@@ -1003,24 +1003,7 @@ static void ExistsSync(const FunctionCallbackInfo<Value>& args) {
   THROW_IF_INSUFFICIENT_PERMISSIONS(
       env, permission::PermissionScope::kFileSystemRead, path.ToStringView());
 
-  uv_fs_t req;
-  auto make = OnScopeLeave([&req]() { uv_fs_req_cleanup(&req); });
-  FS_SYNC_TRACE_BEGIN(access);
```

Comment:
Removing this would break tracing

Reply:
Ah, true. I'll surround `std::filesystem::exists` call with it.